app/login/SubmitLoginForm.tsx

```diff
@@ -0,0 +1,31 @@
+export interface LoginFormData {
+  email: string;
+  password: string;
+  cfTurnstileResponse: string;
+}
+
+type LoginFormResponse = any;
```

Comment:
lets try not to use any.
you can see the response type from the api documentation